footylib/footylib.py

```diff
@@ -20,17 +21,17 @@ class Footy(object):
     def __init__(self):
         self.logger = logging.getLogger('{base}.{suffix}'.format(
             base=LOGGER_BASENAME, suffix=self.__class__.__name__))
-        self.site = "http://www.footy.eu/"
-        self.headers = {'User-Agent': 'Mozilla/5.0'}
+        self._site = "http://www.footy.eu/"
+        self._headers = {'User-Agent': 'Mozilla/5.0'}
```

Comment:
Thats fine, use a variable. It just does not need to be attached to self since it is not used anywhere else.